doc/classes/Viewport.xml

```diff
@@ -128,6 +128,13 @@
 				Returns the [Control] having the focus within this viewport. If no [Control] has the focus, returns null.
 			</description>
 		</method>
+		<method name="gui_get_mouse_over" qualifiers="const">
+			<return type="Control" />
+			<description>
+				Returns the [Control] that the mouse is currently hovering over in this viewport. If no [Control] has the cursor, returns null.
+				This value is always [Control] which is the has the mouse, typically the leaf node or deepest level of the tree which claims focus. This is very useful when used together with [method Node.is_ancestor_of] to find if the mouse is within a control tree.
```

Comment:
> which claims focus

In regard to Control nodes, "focus" has the meaning, that is described for example on on this docs page: https://docs.godotengine.org/en/latest/tutorials/ui/gui_navigation.html

The Godot concept of Control node focus is unrelated to the mouse-over state. In this context, I would consider this wording ambiguous and would prefer, if "focus" is not mentioned here.

Reply:
Tried reworking the doc string, is that better?